src/components/atoms/Badge/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './Badge.js'
```

Comment:
Remove the .js from here:

it should be just
`export { default } from './Badge`